packages/generate/lib/commands/project.js

```diff
@@ -143,6 +153,33 @@ After the first run the token gets stored in your system's keychain and will be
 			validate: validateAlphanumericDash,
 			filter: ( value ) => value.replace( '.required.test', '' ),
 		},
+		{
+			type: 'input',
+			name: 'productionHostAliases',
+			message: 'Enter comma-separated list of production hostname aliases (example.ch,example.de):',
```

Comment:
For a better UX I'd use a recursive question here, see [this example](https://github.com/SBoudrias/Inquirer.js/blob/0053e3f5694a4f75c4901512ab87e8906d1d7896/packages/inquirer/examples/recursive.js). This would also allow to validate each hostname on it's own so you'd know which hostname might me wrong.